### PR TITLE
EN_Reboot : Translate main story quest 

### PR DESCRIPTION
**Pull request by calemyu**
Translated main story quest and put the files in a new folder `/Translated/Quest/Main Story`
Haven't translated the summary of the quest

### CODE REVIEW

File: Translated/Quests/Main Story/ms0010100101.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Title#0,"""First Trial"""
+Summary#0,"""セントラルシティのリーダーである<br>クロフォードと出会った<br><%me>は、<br>エアリオタウンを壊滅させたのが、<br>“ダークファルス”というドールズだと知る。<br><br>ケガの癒えたアイナも加わり、<br>３人はアークスとしての戦い方を学び始める。"""
+text0101#0,"""Head to the \""First Step\"" Cocoon """
+text0102#0,"""Enter to \""First Step\"" Cocoon """
```
Comment (DabirSA) [inline Translated/Quests/Main Story/ms0010100101.csv]: No spaces at ends of lines please

File: Translated/Quests/Main Story/ms0010100104.csv [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Title#0,"""Oranje's Advice"""
+Summary#0,"""次に、オペレーターのオランジェから、<br>フードスタンドの利用方法を教わる。<br><br>“クイックフード”を作るために、<br>まず、セントラルシティの周辺に棲息する<br>テイムズから肉を入手する。"""
+text0101#0,"""Talk to Oranje"""
+text0102#0,"""Gather Tames Meat"""
+text0103#0,"""Slay Dian to obtain シャキリッタ肉"""
+text0104#0,"""Slay Sunny to obtain マタリーナ肉"""
+text0105#0,"""Slay Patt to obtain サッパリーナ肉"""
+text0106#0,"""Slay Moobel to obtain ガッツリーナ肉"""
```
Comment (DabirSA) [inline Translated/Quests/Main Story/ms0010100104.csv]: We have names for the meat now. In order, these will be "Sharp Meat", "Nourishing Meat", "Refreshing Meat" and "Healthy Meat". I don't know why they're slightly different here from the actual item names.

File: Translated/Quests/Main Story/ms0010100105.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Title#0,"""Rossa's Advice"""
+Summary#0,"""続いて、オペレーターのロッサから、<br>セントラルシティの“アイテムラボ”で、<br>装備品を強化する方法を教わる。"""
+text0101#0,"""Talk to Rossa"""
+text0102#0,"""Enchance any weapon to +1 or higher """
```
Comment (DabirSA) [inline Translated/Quests/Main Story/ms0010100105.csv]: Please fix space at end of line

File: Translated/Quests/Main Story/ms0010100107.csv [hunk 4]
```diff
@@ -1,8 +1,8 @@
-Title#0,"""マグナス山のドールズ"""
+Title#0,"""DOLLS in Mt. Magnus"""
 Summary#0,"""戦いの準備を整えた<br><%me>は、<br>アイナたちとともに、<br>マグナス山に出現したドールズの討伐に向かう。"""
-text0101#0,"""マグナス山のふもとでマノンと話す"""
-text0102#0,"""山頂に向かう"""
-text0103#0,"""ナグルスを倒す"""
+text0101#0,"""Talk to Ran at the foot of Mt.Magnus"""
```
Comment (DabirSA) [inline Translated/Quests/Main Story/ms0010100107.csv]: Need a space in "Mt. Magnus"

File: Translated/Quests/Main Story/ms0010100108.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Title#0,"""Clearing the Cocoon"""
+Summary#0,"""クロフォードは、<br><%me>に、<br>“コクーン”と呼ばれる<br>訓練場に向かうように言う。<br><br>その施設では、アークスとしての基本技能を<br>体得することができるのだという。"""
+text0101#0,"""Clear the \""First Step\" Ccocoon""""
```
Comment (DabirSA) [inline Translated/Quests/Main Story/ms0010100108.csv]: Spelling on "Cocoon"

File: Translated/Quests/Main Story/ms0010100121.csv [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Title#0,"""Citizens of Central City"""
+Summary#0,"""アイナの告白に、戦いの決意を新たにした<br><%me>たち。<br><br>ブルーダーから紹介された<br>セントラルシティの住人たちから、<br>戦う術を教わる。"""
+Objective311#0,"""Talk to designated ARKS member"""
+Objective321#0,"""Head to the Class Counter"""
+Objective331#0,"""Set a Subclass"""
+Objective341#0,"""Accept the Side Tasks from Slona"""
+Objective342#0,"""Accept the Side Tasks from Kartevie """
```
Comment (DabirSA) [inline Translated/Quests/Main Story/ms0010100121.csv]: Line ending space again

**Review by DabirSA — CHANGES_REQUESTED** [review]

File: Translated/Quests/Main Story/ms0010100104.csv (hunk 2 above)
Comment (calemyu) [reply Translated/Quests/Main Story/ms0010100104.csv]: Should i change "Slay" to "Subdue" ?

**Comment by calemyu** [comment]
Thanks for reviewing the code! Fixed it all

**Review by DabirSA — APPROVED** [review]

**Pull request merged**